README.md

```diff
@@ -1 +1,102 @@
-# vespa-clone-frontend
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
- [ ] Your front end is very beautiful.  Kindly, add a screenshot to show it off to your users. 💪